code/mergeblock/merge.py

```diff
@@ -0,0 +1,74 @@
+#Jakub Podolak for Plastic Monkeys Cansat 2019
+
+import subprocess
+import configparser
+import msgpack
+import time
+
+#our command to get one last line from file
+tail_command = "tail -n 1 "
+
+#initialize config parser, read from config.ini
+config = configparser.ConfigParser()
+config.read('config.ini')
+
+#read some settings from config file
+output_path = config['SETTINGS']['output']
+datalog_path = config['SETTINGS']['datalog']
+sleep_time = float(config['SETTINGS']['sleeptime'])
+
+#function to read paths to sensors' logs from config file
+def getDirectories():
+    directories = []
+    paths = config.items("PATHS")
+    for key, path in paths:
+        directories.append(path)
+    return directories
+
+#function takes paths to logs as an argument, returns list of the latest readings(int) from all paths
+def getTails(directories):
+    tails = []
+    for x in directories:
+        command = tail_command + x
+        result = subprocess.Popen(command.split(), stdout=subprocess.PIPE).communicate()[0]
+        result.rstrip()
+        s = str(result, 'utf-8')
+        number = int(s)
+        tails.append(number)
+    return tails
+
+#function takes list of latest readings (ints) and returns packet in readable form
+def prepareTextPack(tails):
```

Comment:
I don't really understand why do we need to prepare text from that, message pack works well with standard lists of values, so there is no need to stringify that

Reply:
I just leave this as a quick gate to debug, easy to turn it off later

Reply:
I've added to config file option to turn it off

Reply:
if its a debug feature, it should be turned off by default

Reply:
So by default we set this feature to false in our config files. That's not a big deal really